client/packages/system/src/RequestRequisitionLine/operations.graphql

```diff
@@ -0,0 +1,73 @@
+fragment ItemWithStats on ItemNode {
+  id
+  name
+  code
+  unitName
+  defaultPackSize
+  availableStockOnHand(storeId: $storeId)
+  stats(storeId: $storeId) {
+    averageMonthlyConsumption
+    availableStockOnHand
+    availableMonthsOfStockOnHand
+    totalConsumption
+    stockOnHand
+    monthsOfStockOnHand
+  }
+}
+
+fragment itemInformation on RequisitionItemInformationNode {
+  id
+  adjustmentsInUnits
+  amcInUnits
+  name(storeId: $storeId) {
+    id
+    name
+  }
+  outgoingUnits
+  dateRange
+  stockInUnits
+}
+
+fragment RequestLine on RequisitionLineNode {
+  id
+  itemId
+  requestedQuantity
+  suggestedQuantity
+  comment
+  itemName
+  requisitionNumber
+  initialStockOnHandUnits
+  incomingUnits
+  outgoingUnits
+  lossInUnits
+  additionInUnits
+  expiringUnits
+  daysOutOfStock
+
+  itemStats {
+    __typename
+    availableStockOnHand
+    availableMonthsOfStockOnHand
+    averageMonthlyConsumption
+  }
+  linkedRequisitionLine {
+    approvedQuantity
+    approvalComment
+  }
+  itemInformation {
+    __typename
+    ...itemInformation
+  }
+  item {
+    ...ItemWithStats
+  }
+  reason {
+    ...ReasonOptionRow
+  }
+}
+
+query OnlyHereToAvoidUnusedWarnings {
```

Comment:
had to add this otherwise wasn't compiling because getSdk was empty and not using parameters